geopandas/tests/test_plotting.py

```diff
@@ -871,6 +872,47 @@ def test_plot(self):
         self.df.plot()
 
 
+class TestColorParamArray:
+    def setup_method(self):
+        geom = []
+        color = []
+        for a, b in zip(np.linspace(0, 10, 4)[::2], np.linspace(0, 10, 4)[1::2]):
```

Comment:
```suggestion
        for a, b in [(0, 2), (4, 6)]:
```

I suppose the exact numbers don't matter? (then this is a bit simpler to read)